packages/transformers/posthtml/src/loadPlugins.js

```diff
@@ -0,0 +1,46 @@
+// @flow strict-local
+
+import type {FilePath} from '@parcel/types';
+import localRequire from '@parcel/local-require';
+
+export default async function loadExternalPlugins(
+  plugins: Array<string> | {+[pluginName: string]: mixed},
+  relative: FilePath
+): Promise<Array<mixed>> {
+  if (Array.isArray(plugins)) {
+    return Promise.all(
+      plugins.map(p => loadPlugin(p, relative)).filter(Boolean)
+    );
+  } else if (typeof plugins === 'object') {
+    let _plugins = plugins;
+    let mapPlugins = await Promise.all(
+      Object.keys(plugins).map(p => loadPlugin(p, relative, _plugins[p]))
+    );
+    return mapPlugins.filter(Boolean);
+  } else {
+    return [];
+  }
+}
+
+async function loadPlugin(
+  pluginArg: string | Function,
+  relative: FilePath,
+  options: mixed = {}
+): mixed {
+  if (typeof pluginArg !== 'string') {
```

Comment:
I took this whole file from #3037, just had to add this line in case the config had a require statement. As @wbinnssmith mentioned, we could find a common place for this to live. Until then I duplicated it.